### PR TITLE
Log GL_MAX_VERTEX_ATTRIBS value before making programs

### PR DESCRIPTION
**Pull request by anandthakker**
Temporary diagnostic logging for https://github.com/mapbox/mapbox-gl-native/issues/8150

### CODE REVIEW

**Comment by anandthakker** [comment]
@zugaldia still working on getting my Android build working, but I confirmed that it logs `GL_MAX_VERTEX_ATTRIBS = 16` to the console when I run the glfw test app.

File: src/mbgl/gl/context.cpp [hunk 1]
```diff
@@ -37,10 +38,16 @@ static_assert(underlying_type(TextureFormat::Alpha) == GL_ALPHA, "OpenGL type mi
 Context::~Context() {
     reset();
 }
+    
+void Context::logGlMaxVertexAttribsValue() {
+    int glMaxVertexAttribs;
+    glGetIntegerv(GL_MAX_VERTEX_ATTRIBS, &glMaxVertexAttribs);
```
Comment (mb12) [inline src/mbgl/gl/context.cpp]: Is it possible to wrap this call under MBGL_CHECK_ERROR macro. Its helpful to catch GL bugs. Its stripped out in release builds.

**Comment by zugaldia** [comment]
Just confirming that I can see the message in the Android Studio logs:

image

File: src/mbgl/gl/context.cpp (hunk 1 above)
Comment (anandthakker) [inline src/mbgl/gl/context.cpp]: 👍 thanks @mb12

**Comment by jfirebaugh** [comment]
I think we're good here; we got the information we need, which is that some implementations have a max of 8.

**Pull request closed without merging**